ddl/ddl_worker.go

```diff
@@ -952,15 +954,21 @@ func (w *worker) runDDLJob(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64,
 	}()
 	if job.IsFinished() {
 		logutil.Logger(w.logCtx).Debug("[ddl] finish DDL job", zap.String("job", job.String()))
-		return
+		return 0, nil
```

Comment:
This changed semantic, before return ver, err,  Now is 0, nil.

Reply:
ok